share/spack/gitlab/cloud_pipelines/stacks/aws-pcluster-neoverse_n1/spack.yaml

```diff
@@ -0,0 +1,61 @@
+spack:
+  view: false
+
+  definitions:
+  - compiler_specs:
+    - gcc
```

Comment:
So is this stack building gcc, or does it already exist in the container?

Reply:
There is a gcc in the container to build all the other apps.
The stack builds gcc for the buildcache to be used as compiler on the user's systems.

Reply:
The latest pipeline picks up the correct container. There is a `gcc@12.3.0` pre-installed to build all the software.

But our ParallelCluster users will want to build software themselves. Forthat they want a recent gcc. That's the `gcc@12.3.0%gcc@7.3.1` which is made available through his pipeline. 

I hope this clarifies what I am trying to achieve.

Reply:
yup, that makes sense, thanks!